docs/walrus-sites/tutorial-publish.md

```diff
@@ -9,16 +9,25 @@ The `site-builder` works by uploading a directory of files produced by any web f
 and adding the relevant metadata to Sui. This directory should have a file called `index.html` in
 its root, which will be the entry point to the Walrus Site.
 
-For the rest of the tutorial, we will use as an example the simple site contained in
-`./examples/snake`.
+There is a very useful [example-Walrus-sites](https://github.com/MystenLabs/example-walrus-sites)
+repository that contains multiple kinds of sites that you can use for reference.
+
+For simplicity, we will start by publishing the most frugal of the sites, the `walrus-snake` game.
+
+First, clone the repository of examples:
+
+``` sh
+git clone https://github.com/MystenLabs/example-walrus-sites.git && cd walrus-snake/
+```
```

Comment:
Important: we should point to `./examples/snake` in the walrus sites repo, which is now different from the `walrus-snake`. In examples/snake we also show `ws-config.json`.

Reply:
Wouldn't it be better to update the `walrus-snake` instead? 

I think it would be beneficial promoting the walrus sites examples repo to the community, in order for them to try out more sites after they complete the tutorial.

Reply:
PR that updates the `walrus-snake`: https://github.com/MystenLabs/example-walrus-sites/pull/15